docs/docs/administration/reverse-proxy.md

```diff
@@ -64,3 +64,73 @@ Below is an example config for Apache2 site configuration.
    ProxyPreserveHost On
 </VirtualHost>
 ```
+
+### Traefik Proxy example config
+
+The example below is for Traefik version 3.
+
+The most important is to increase the `respondingTimeouts` of the entrypoint used by immich. In this example of entrypoint `websecure` for port `443`. Per default it's set to 60s which leeds to videos stop uploading after 1 minute (Error Code 499). In the example it's set to 10 minutes. Hence video-uploads will fail after 10 minutes instead of one.
+
+`traefik.yaml`
+```yaml
+[...]
+entryPoints:
+  websecure:
+    address: :443
+    AsDefault: true
+    http:
+      tls:
+        certresolver: cloudflare-resolver
+        domains:
+          - main: "*.domain.com"
+            sans: 
+              - "domain.com"
```

Comment:
Is this section required? The other examples just show a plain http setup.

Reply:
No, for http it's not required. I'll remove it